admin_app/src/app/dashboard/page.tsx

```diff
@@ -64,7 +69,9 @@ const Dashboard: React.FC = () => {
                 {dashboardPage}
               </Typography>
             </Box>
-            <TabPanel tabValue={timePeriod} handleChange={handleTabChange} />
+            {shouldShowTabPanel() && (
```

Comment:
Check out [Tao of React](https://alexkondov.com/tao-of-react/#conditional-rendering). Let's use `?` instead of `&&`